associated-token-account/program/src/instruction.rs

```diff
@@ -34,6 +34,24 @@ pub enum AssociatedTokenAccountInstruction {
     ///   4. `[]` System program
     ///   5. `[]` SPL Token program
     CreateIdempotent,
+    /// Transfers from and closes a nested associated token account: an
+    /// associated token account owned by an associated token account.
+    ///
+    /// The tokens are moved from the nested associated token account to the
+    /// wallet's associated token account.
```

Comment:
nit: document where the rent reserve for the nested token account ends up (presumably in the owner's account)